ironmon_tracker/constants/MoveData.lua

```diff
@@ -4966,7 +4966,7 @@ MoveData.MOVES_MASTER_LIST = {
 		pp = "15",
 		accuracy = "100",
 		category = MoveData.MOVE_CATEGORIES.PHYSICAL,
-		description = "Inflicts regular damage. Removes the target's immunity to ground-type damage."
+		description = "Inflicts regular damage. Removes the target's immunity to ground-type damage. This move will hit targets under the effect of bounce, fly, or sky drop."
```

Comment:
For consistent naming across other descriptions, the moves should be capitalized (Bounce, Fly, or Sky Drop)